changelog.txt

```diff
@@ -0,0 +1 @@
+feature - Use signed upload URL when uploading functions code.

```

Comment:
This isn't really a feature for users. I'd maybe do something like:

    fixed - Updated functions upload process to be more future-proof.